### PR TITLE
Add tests for realm in target argument for "script.evaluate", "script.callFunction" and "script.disown".

### PR DESCRIPTION
**Pull request by lutien**
Add tests for `realm` in target argument for "script.evaluate", "script.callFunction" and "script.disown".

These tests were already reviewed in mozilla repo, when approved here, the tests will be synced from mozilla repo.

### CODE REVIEW

**Comment by whimboo** [comment]
@sadym-chromium if you could find some time to review this PR we would appreciate. It's the last feature that we need to finally be able to enable the script module by default and we would like to have it in the first 206.0b1 release (which requires to land this week). If doable we would kinda appreciate!

**Comment by lutien** [comment]
Hey @sadym-chromium, would you have a minute to have a look at these tests? If not, let us know, so we can then land them in mozilla repo and follow up later.

**Comment by sadym-chromium** [comment]
Sure, I'm on it right now

File: webdriver/tests/bidi/script/call_function/this.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+@pytest.mark.asyncio
+async def test_default_this(bidi_session, top_context):
+    result = await bidi_session.script.call_function(
+        function_declaration="function(){return this}",
+        await_promise=False,
+        target=ContextTarget(top_context["context"]))
+
+    # Note: https://github.com/w3c/webdriver-bidi/issues/251
```
Comment (sadym-chromium) [inline webdriver/tests/bidi/script/call_function/this.py:35]: Is the comment relevant?

File: webdriver/tests/bidi/script/evaluate/evaluate.py [hunk 2]
```diff
@@ -24,3 +26,73 @@ async def test_interact_with_dom(bidi_session, top_context):
     assert result == {
         "type": "string",
         "value": "window.location.href: about:blank"}
+
+
+@pytest.mark.asyncio
+async def test_target_realm(bidi_session, top_context):
+    realms = await bidi_session.script.get_realms(context=top_context["context"])
+    default_realm = realms[0]["realm"]
```
Comment (sadym-chromium) [inline webdriver/tests/bidi/script/evaluate/evaluate.py:34]: maybe add a fixture for the default_realm?

**Review by sadym-chromium — APPROVED** [review]

File: webdriver/tests/bidi/script/evaluate/evaluate.py (hunk 2 above)
Comment (lutien) [reply webdriver/tests/bidi/script/evaluate/evaluate.py]: Good idea! Added it

File: webdriver/tests/bidi/script/call_function/this.py (hunk 1 above)
Comment (lutien) [reply webdriver/tests/bidi/script/call_function/this.py]: Looks like not anymore, removed the comment

**Comment by lutien** [comment]
These changes will be synced from the mozilla repo

**Comment by whimboo** [comment]
FYI the upstream merge will be done via https://github.com/web-platform-tests/wpt/pull/35962.

**Pull request closed without merging**